src/components/calcite-radio-group-item/calcite-radio-group-item.scss

```diff
@@ -1,101 +1,93 @@
 :host {
-  display: flex;
-  align-self: stretch;
-  cursor: pointer;
+  @apply flex self-stretch cursor-pointer;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
 }
 
 :host label {
-  display: flex;
-  flex: 1;
-  color: var(--calcite-ui-text-3);
+  @apply flex 
+    text-color-3 
+    m-1 
+    box-border 
+    pointer-events-none 
+    items-center
+    flex-grow;
   border: 1px solid transparent;
```

Comment:
`border border-solid` + `border-color: transparent` (should we add border-color-transparent ?)